src/frontend/flight/sync/writer.rs

```diff
@@ -380,6 +410,12 @@ impl SeafowlDataSyncWriter {
         self.remove_sync(&url);
         self.advance_durable();
 
+        // Record flush metrics
+        let flush_duration = start.elapsed().as_millis();
+        self.metrics.flushing_time.record(flush_duration as f64);
+        self.metrics.flushed_size(size as u64);
+        self.metrics.flushed_rows(rows as u64);
+
         Ok(())
```

Comment:
Let's also add a gauge here for timestamp of last flush

```
seafowl_writer_last_successful_flush_timestamp_seconds
```

Reply:
Good idea, adding.